tests/test_optimizely_config.py

```diff
@@ -72,7 +72,7 @@ def setUp(self):
                     },
                     'id': '111133',
                     'key': 'test_experiment2',
-                    'audiences': ''
+                    'audiences': None
```

Comment:
None makes sense here, but let's use empty string for consistency  as defined in the spec.

Reply:
Corrected